blogposts/announcing-sourcegraph-3.8.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Sourcegraph 3.8: Introducing campaign management with Automation and LSIF-based code intelligence"
+author: Christina Forney
+publishDate: 2019-09-01T10:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-3.8
+heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
+published: true
+---
+
+Sourcegraph is the standard developer platform for code search and navigation at many of the largest and most exacting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
```

Comment:
Is this a typo or did you actually mean exacting?

```suggestion
Sourcegraph is the standard developer platform for code search and navigation at many of the largest and most exciting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
```

Reply:
I actually don't know what was originally intended here - @ryan-blunden  do you remember?

Reply:
I am going to withdraw my suggested change and assume "exacting" was the intended meaning, but I do think "exacting" is a bit of an unusual word. What about "advanced"?

Reply:
+1 will change!